src/controllers/commands/CommandController.cpp

```diff
@@ -637,6 +638,37 @@ void CommandController::initialize(Settings &, Paths &paths)
             return "";
         });
 
+    this->registerCommand("/popout", [](const QStringList &words,
+                                        ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            if (!channel->isTwitchChannel() || channel->isEmpty())
```

Comment:
Kind of nitpicky since the `CommandController` handles this for us, but we could check whether `channel` actually has a name associated (like with https://github.com/Chatterino/chatterino2/pull/2528).